fastly.toml

```
@@ -1,5 +1,12 @@
-manifest_version = 2
-name = "Empty starter for JavaScript"
-description = "An empty application template for the Fastly Compute@Edge environment which simply returns a 200 OK response."
+# This file describes a Fastly Compute@Edge package. To learn more visit:
+# https://developer.fastly.com/reference/fastly-toml/
+
 authors = ["<oss@fastly.com>"]
+description = "An empty application template for the Fastly Compute@Edge environment which simply returns a 200 OK response."
 language = "javascript"
+manifest_version = 2
+name = "Empty starter for JavaScript"
+service_id = ""
+
+[scripts]
+  build = "npm exec webpack && npm exec js-compute-runtime ./bin/index.js ./bin/main.wasm"
```

These changes are unnecessary.

```suggestion
manifest_version = 2
name = "Empty starter for JavaScript"
description = "An empty application template for the Fastly Compute@Edge environment which simply returns a 200 OK response."
```